src/handlebars/archive.handlebars

```diff
@@ -60,16 +70,16 @@
                           \{{else}}
                             <td></td>
                           \{{/if}}
-                          <td class='download-td'>
-                            <a class='blue-button no-underline' href='\{{link}}'><var binary-info>\{{type}} (\{{size}} MB)</var></a>
-                          </td>
                           \{{#if installer_link}}
                           <td class='download-td'>
-                            <a class='blue-button no-underline' href='\{{installer_link}}'><var binary-info>Installer \{{type}}</var></a>
+                            <a class='blue-button no-underline' href='\{{installer_link}}'><var binary-info>\{{type}} (\{{installer_extension}})</var></a>
```

Comment:
Looking at staging this creates a bit of an inconsistency.  e.g. It shows `.pkg` if there's an installer but doesn't show `tar.gz` if it's a straight binary

Reply:
removed